### PR TITLE
Optimal stopping point

### PR DESCRIPTION
**Pull request by hemajv**
## Related Issues and Dependencies

#336 

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

This notebook looks into different distribution types observed for TestGrid tests to determine a point after which the test has a higher probability of failing

### CODE REVIEW

File: notebooks/data-sources/TestGrid/metrics/time_to_fail.ipynb [hunk 1]
```diff
@@ -8,9 +8,7 @@
     "\n",
```
[the remark this answers is not in the document]
Comment (hemajv) [reply notebooks/data-sources/TestGrid/metrics/time_to_fail.ipynb:8]: <p>yes its an updated dataset, we will add it to the repo </p>

[the remark this answers is not in the document]
Comment (hemajv) [reply notebooks/data-sources/TestGrid/metrics/time_to_fail.ipynb:8]: added a description on the parameters

[the remark this answers is not in the document]
Comment (oindrillac) [reply notebooks/data-sources/TestGrid/metrics/time_to_fail.ipynb:8]: thanks!

**Review by oindrillac — COMMENTED** [review]
/lgtm

**Pull request merged**